CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ We use this CHANGELOG to document breaking changes, new features, bug fixes, and
 
 ## 2025
 
+* TBD: Configuration resolved at run time is now written to a file `results/run_config.yaml`. @victorlin
```

Comment:
Small nit

```suggestion
* TBD: Configuration resolved at run time is now written to a file `results/{build}/run_config.yaml`. @victorlin
```

Reply:
Added with README changes in 699daa371978a7f20d842e39a318eb3ae2eb99a6